.github/workflows/mirror_gitee.yml

```
@@ -15,5 +15,5 @@ jobs:
           ORGANIZATION: deepmodeling
           SSH_PRIVATE_KEY: ${{ secrets.SYNC_GITEE_PRIVATE_KEY }}
         with:
-          source-repo: "git@github.com:deepmodeling/dpgen.git"
+          source-repo: "http://github.com/deepmodeling/dpgen.git"
```

use https instead of http
```suggestion
          source-repo: "https://github.com/deepmodeling/dpgen.git"
```